app/lib/searchValidator.js

```diff
@@ -4,7 +4,7 @@ function validateSearch(search) {
   let errorMessage = null;
   let searchToReturn = search;
 
-  if (!search) {
+  if ((!search) || ((search) && (search.trim() === ''))) {
```

Comment:
Maybe just `if (search && search.trim() !== "" )` to give a less complicated condition and reverse the if/else statements?

Even better make intent explicit by pulling the condition out into a function `if (notEmpty(search)) ..` then you can reuse it below.